packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -7,39 +7,78 @@ import { ITelemetryLogger, ITelemetryBaseEvent } from "@fluidframework/common-de
 import { TelemetryLogger } from "./logger";
 
 /**
- * The MockLogger records events sent to it, and then can walk back over those events
+ * Logger that records events sent to it, and then can walk back over those events
  * searching for a set of expected events to match against the logged events.
+ *
+ * @internal
  */
 export class MockLogger extends TelemetryLogger implements ITelemetryLogger {
+	/**
+	 * Internal buffer of logged events.
+	 * It's recommended to use the methods on this class to validate events stored here.
+	 *
+	 * @remarks
+	 * Gets cleared when any of the validation functions on this class gets called.
+	 *
+	 * @internal
+	 */
 	events: ITelemetryBaseEvent[] = [];
 
+	/**
+	 * @internal
+	 */
 	constructor() {
 		super();
 	}
 
+	/**
+	 * Clears {@link MockLogger.events}.
+	 * @internal
```

Comment:
I don't think the per-member `@internal` tags are necessary. If the type itself is internal, I think the implication is pretty clear.

Reply:
I did this following the advice I've seen in other places for deprecations, but I think you're right this isn't as helpful... there'll be no strikeout or any visual cue on the method calls like a deprecated method would have, to provide immediate indication that the parent is deprecated (or in this case internal). And I did notice that api-extractor doesn't add any annotation to the methods. I can clean up the per-method `@internal` annotations.